examples/IRGCTCPServer/IRGCTCPServer.ino

```diff
@@ -49,7 +53,7 @@ WiFiServer server(4998);  // Uses port 4998.
 WiFiClient client;
 
 uint16_t *code_array;
-IRsend irsend(4);  // an IR emitter led is connected to GPIO pin 4
+IRsend irsend(4);  // An IR emitter led is connected to GPIO pin 4
```

Comment:
This could be  `// An IR LED is controlled by GPIO pin 4 (D2)` as in IRGCSendDemo.ino